### PR TITLE
Add dropdown menu to Config

### PR DESCRIPTION
**Pull request by stijndcl**
Added a dropdown menu to Config to change the submission platform to Naos or Dodona without having to change the link in the code every time

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,13 @@
           "default": null,
           "description": "Specifies which API token should be used to authorize to Dodona.",
           "scope": "application"
+        },
+        "dodona.platform": {
+          "type": "string",
+          "default": "Dodona",
+          "enum": ["Dodona", "Naos"],
```
Comment (thepieterdc) [inline package.json:47]:
```suggestion
          "default": "https://dodona.ugent.be",
          "enum": ["https://dodona.ugent.be", "https://naos.ugent.be"],
```

File: package.json [hunk 2]
```diff
@@ -40,6 +40,13 @@
           "default": null,
           "description": "Specifies which API token should be used to authorize to Dodona.",
           "scope": "application"
+        },
+        "dodona.platform": {
```
Comment (thepieterdc) [inline package.json:44]:
```suggestion
        "dodona.api.host": {
```

File: package.json [hunk 3]
```diff
@@ -40,6 +40,13 @@
...
+          "type": "string",
+          "default": "Dodona",
+          "enum": ["Dodona", "Naos"],
+          "description": "Specifies whether submissions should be sent to Dodona or Naos.",
```
Comment (thepieterdc) [inline package.json:48]:
```suggestion
          "description": "Specifies which Dodona instance should be used.",
```

File: src/extension.ts [hunk 4]
```diff
@@ -4,16 +4,28 @@ import { identify } from "./exercise/identification";
...
+            // Get the configuration from the settings.
+            // Re-declare in case the user changed it.
+            config = vscode.workspace.getConfiguration('dodona');
+
+            // Declare this here so that the platform can be changed without restarting.
+            // Cast to lowercase so the enum values can be CamelCased to look better.
+            // Error can be ignored as this configuration has a default value & is always a string.
+            //@ts-ignore
+            const domain = config.get("platform").toLowerCase();
+            const post = bent(`https://${domain}.ugent.be`, 'POST', 'json');
```
Comment (thepieterdc) [inline src/extension.ts:27]: By specifying the host in the settings directly (see previous remark), the domain does not need to be lowercased and can be used directly.
**Make sure to test if it still works, it's possible that `@ts-ignore` is still required anyway**

```suggestion
            const post = bent(config.get('api.host'), 'POST', 'json');
```

**Review by thepieterdc — CHANGES_REQUESTED** [review]

**Comment by stijndcl** [comment]
Tested, still works as expected

**Review by thepieterdc — APPROVED** [review]

**Pull request merged**